src/main/java/teammates/logic/core/InstructorsLogic.java

```diff
@@ -174,6 +182,14 @@ public void verifyIsEmailOfInstructorOfCourse(String instructorEmail, String cou
         }
     }
 
+    public void verifyAtleastOneInstructorIsDisplayed(String courseId, boolean isEditedInstructorDisplayed)
+            throws InvalidParametersException {
+
+        if (isNoInstructorDisplayedToStudents(courseId) && !isEditedInstructorDisplayed) {
```

Comment:
Not sure if I follow the logic here. What's the condition you propose here that ascertains that "at least one instructor is displayed"?

Reply:
I just found a bug in this implementation, will update you on the logic once I have completed fixing it.